### ISSUE TITLE
Enabling extra modules

### ISSUE THREAD

**Issue by m-byte** [UrNVR/urnvr#3]
Currently, this project uses the default module selection from Ubiquiti. While this is a good starting point as it is known to include any necessary modules, there are some strange choices (e.g. drivers for WiFi on a device with only wired network and Bluetooth connectivity) that might be sensible to clean up. At the same time, this selection does not include all modules required to run e.g. docker. For now, users have to use `make kernel-menuconfig` to enable any modules they need. However, a sensible default selection would go a long way toward making this easier to use.

If you know of any modules that you'll need, feel free to list them here or to create a PR.

**Comment by m-byte** [comment]
Obviously, even discussing potential uses of the UNVR can be helpful, as the necessary modules also depend on what the machine will be used for.

E.g. I will be using mine as router, NAS and phone system (i.e. essentially a general purpose home server), so I enabled modules required for docker and a lot of the netfilter stuff.

**Comment by chrischsc** [comment]
would you provide the config you use? For sure interesting is tun.ko and crypto support for wireguard...

**Comment by cainmp** [comment]
For me I want two main things:
Docker (or podman) support
Wireguard kernel module support
